components/List/list.css

```diff
@@ -0,0 +1,80 @@
+.unorder-container {
+    height: 11rem;
+    width: 33rem;
+    border: 2px solid black;
+    margin-bottom: 1rem;
+}
+
+.unorderlist {
+    display: flex;
+    justify-content: center;
+    margin-top: 2rem;
+    font-size: 1.5rem;
+}
+
+.orderlist {
+    display: flex;
+    margin-top: 5rem;
+    font-size: 1.5rem;
+}
+
+.orderlist-container {
+    margin-bottom: 5rem;
+    height: 18rem;
+    width: 48rem;
+    border: 2px solid black;
+    display: flex;
+    flex-direction: column;
+    /*margin-bottom: 1rem ;*/
+}
+
+.list-style-disc {
+    list-style-type: disc;
+    margin-right: 2rem;
+}
+
+.list-style-circle {
+    margin-right: 2rem;
+    list-style-type: circle;
+}
+
+.list-style-square {
+    margin-right: 2rem;
```

Comment:
you have given same margin, what you can do you can make a class with any name and give that margin right : 2rem and then add that class to every element where that margin is required, we have to follow DRY principle(do not repeat yourself).

Reply:
okkk thanks for your suggestion

Reply:
okkk nice suggestion